Samples/TollApp/VSProjects/TollApp/Utils/AzureResourcesCreator.cs

```
@@ -0,0 +1,74 @@
+
+                //Create a new container, if it does not exist
+                container.CreateIfNotExists();
+
+                using (var fileStream = File.OpenRead(@"registration.json"))
+                {
+                    registrationBlockBlob.UploadFromStream(fileStream);
+                }
+
+                //read Registration.json from BLOB to be used for random data generator
+                using (var stream = new MemoryStream())
+                {
+                    registrationBlockBlob.DownloadToStream(stream);
+                    stream.Position = 0; //resetting stream's position to 0
+                    var serializer = new JsonSerializer();
+
+                    using (var sr = new StreamReader(stream))
+                    {
+                        using (var jsonTextReader = new JsonTextReader(sr))
+                        {
+                            var jsonStream = serializer.Deserialize(jsonTextReader);
+                            return JsonConvert.DeserializeObject<Registration[]>(jsonStream.ToString());
+                        }
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine(ex.ToString());
+                return null;
+            }
+        }
+
+        public static async void CreateAzureCosmosDb()
+        {
+            try
+            {
+                var client = new DocumentClient(new Uri(CloudConfiguration.DocumentDbUri), CloudConfiguration.DocumentDbKey);
+                await client.CreateDatabaseIfNotExistsAsync(new Database {Id = CloudConfiguration.DocumentDbDatabaseName});
+
+                await client.CreateDocumentCollectionIfNotExistsAsync( UriFactory.CreateDatabaseUri(CloudConfiguration.DocumentDbDatabaseName), new DocumentCollection {Id = CloudConfiguration.DocumentDbCollectionName});
+            }
+            catch (Exception ex)
```

why does this continue on failure? Can we remove exception handling and let the program fail in this case?

---

Completed. Not tested.